.rubocop.yml

```diff
@@ -48,7 +48,17 @@ Style/ClassAndModuleChildren:
 Style/GuardClause:
   Enabled: false
 
-Metrics/LineLength:
+# https://rubocop.readthedocs.io/en/latest/cops_style/#stylehasheachmethods
+Style/HashEachMethods:
+  Enabled: true
+# https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashtransformkeys
+Style/HashTransformKeys:
+  Enabled: true
+# https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashtransformvalues
+Style/HashTransformValues:
+  Enabled: true
+
+Layout/LineLength:
```

Comment:
Why the rubocop changes?

Reply:
running `rake ci` outputted below so went ahead and configured them

```
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Style/HashEachMethods
 - Style/HashTransformKeys
 - Style/HashTransformValues
```
https://rubocop.readthedocs.io/en/latest/cops_style/#stylehasheachmethods
https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashtransformkeys
https://rubocop.readthedocs.io/en/latest/cops_style/#stylehashtransformvalues

Reply:
Ok, I see. Was rubocop updated recently or something to cause this?

Reply:
Also, I see that `Metrics/LineLength` was removed and `Layout/LineLength` was added. Was this renamed in a newer version or something?

Reply:
https://github.com/rubocop-hq/rubocop/releases/tag/v0.78.0
```
#7542: (Breaking) Move LineLength cop from Metrics department to Layout department. (@koic)
```

When running `rake ci`
```
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
```

Reply:
removed